quantestpy/simulator/state_vector_circuit.py

```diff
@@ -409,16 +293,32 @@ def _get_whole_gates(self,) -> np.ndarray:
         return whole_gates
 
 
+def cvt_quantestpy_circuit_to_state_vector_circuit(
```

Comment:
This is the converter from `QuantestPyCircuit` to `StateVectorCircuit`. This will be called in the assert methods which assume `StateVectorCircuit` internally.